modules/aws-eks-teams/main.tf

```diff
@@ -145,42 +159,36 @@ resource "aws_iam_role" "team_sa_irsa" {
   tags = var.tags
 }
 
-# ---------------------------------------------------------------------------------------------------------------------
-# Kubernetes Team Service Account
```

Comment:
Same as above, the reason for having this bulky structured comments it to allow users to read between the lines and see as sections ("This is for IRSA", "This is For Kubernetes Service Accounts").
The concept of Teams may be new to some, and many would like to dive deep into the code and understand its structure, having this can help navigate through the code more easily (remember, not everyone are terraform experts).